test/expected/plan_expand_hypertable_optimized.out

```diff
@@ -421,10 +433,242 @@ SELECT * FROM cte ORDER BY value;
+   ->  Append
+         ->  Seq Scan on hyper_w_space
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_103_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_104_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_105_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_106_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_107_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_108_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_109_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_110_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_111_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_112_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_113_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_114_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+         ->  Seq Scan on _hyper_2_115_chunk
+               Filter: (device_id < ANY ('{dev5,dev6}'::text[]))
+(31 rows)
+
+--ALL with equals and different values shouldn't scan any chunks
+:PREFIX SELECT * FROM hyper_w_space WHERE device_id = ALL(ARRAY['dev5','dev6']) ORDER BY value;
+                           QUERY PLAN                            
+-----------------------------------------------------------------
+ Sort
+   Sort Key: hyper_w_space.value
+   ->  Append
+         ->  Seq Scan on hyper_w_space
+               Filter: (device_id = ALL ('{dev5,dev6}'::text[]))
+(5 rows)
+
+--Multi AND
+:PREFIX SELECT * FROM hyper_w_space WHERE time < 10 AND time < 100 ORDER BY value;
+                        QUERY PLAN                        
+----------------------------------------------------------
+ Sort
+   Sort Key: hyper_w_space.value
+   ->  Append
+         ->  Seq Scan on hyper_w_space
+               Filter: (("time" < 10) AND ("time" < 100))
+         ->  Seq Scan on _hyper_2_106_chunk
+               Filter: (("time" < 10) AND ("time" < 100))
+         ->  Seq Scan on _hyper_2_104_chunk
+               Filter: (("time" < 10) AND ("time" < 100))
+         ->  Seq Scan on _hyper_2_105_chunk
+               Filter: (("time" < 10) AND ("time" < 100))
+         ->  Seq Scan on _hyper_2_103_chunk
+               Filter: (("time" < 10) AND ("time" < 100))
+(13 rows)
+
+--Time dimension doesn't filter chunks when used with IN/ANY/ALL
+:PREFIX SELECT * FROM hyper_w_space WHERE time < ANY(ARRAY[1,2]) ORDER BY value;
```

Comment:
time should now work with ANY(one_element) and ALL(one or many). Let's add tests for those cases.